unbxdAutosuggest.js

```diff
@@ -1425,7 +1618,7 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 				+ '{{#data.POPULAR_PRODUCTS}}'
 				+ '<li class="unbxd-as-popular-product ' + (this.options.popularProducts.view === 'grid' ? 'unbxd-as-popular-product-grid' : '')
 				+ '" data-value="{{autosuggest}}" data-index="{{@index}}" data-type="{{type}}" data-pid="{{pid}}" >'
-				+ (this.options.popularProducts.tpl ? this.options.popularProducts.tpl : this.default_options.popularProducts.tpl)
+				+ ((this.options.popularProducts.variants && this.options.popularProducts.variants.tpl) ? this.options.popularProducts.variants.tpl : this.options.popularProducts.tpl ? this.options.popularProducts.tpl : this.default_options.popularProducts.tpl)
 				+ '</li>'
 				+ '{{/data.POPULAR_PRODUCTS}}'
 				+ '{{/if}}';
```

Comment:
Please add as many test cases as possible as it is critical component, At least 10